torch/csrc/jit/codegen/cuda/arith.h

```diff
@@ -184,6 +186,18 @@ TORCH_CUDA_API TensorView* threshold(TensorView* in, Val* thresh, Val* value);
 TORCH_CUDA_API Val* clamp(Val* in, Val* min_val, Val* max_val);
 TORCH_CUDA_API TensorView* clamp(TensorView* in, Val* min_val, Val* max_val);
 
+// Internal operator for supporting backward graphs
```

Comment:
Please convert to doxygen comment (just replace `//` with `//!`)

Reply:
Updated, thanks!